webapp/sources/rudder/rudder-web/src/main/elm/sources/Editor/ViewMethod.elm

```diff
@@ -64,9 +64,10 @@ checkMandatoryParameter methodParam =
              _ -> True
            ) methodParam.constraints
 
-showParam: Model -> MethodCall -> ValidationState MethodCallParamError -> MethodParameter -> CallParameter -> Html Msg
-showParam model call state methodParam param =
+showParam: Model -> MethodCall -> ValidationState MethodCallParamError -> MethodParameter -> List CallParameter -> Html Msg
+showParam model call state methodParam params =
   let
+    displayedValue = List.Extra.find (.id >> (==) methodParam.name ) params |> Maybe.map (.value >> displayValue) |> Maybe.withDefault ""
```

Comment:
We now get the correct value from the list of parameter and not juste hope that they are correctly sorted